css/style.css

```diff
@@ -1,4 +1,914 @@
+  }
+  .data__dimention {
+    align-items: center;
+  }
+  .data__figures {
+    font-size: 51px;
+    padding-right: 6px;
+  }
+  .data__specifications {
+    display: block;
+  }
+  .testdrive__img {
+    display: none;
+  }
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color:#e51937;
+  }
+  .testdrive__text:before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color:#e51937;
+    color: #fff;
+    font-size: 16px;
+  }
+  .arrow {
+    padding-left: 2px;
+  }
+}
+
+@media (min-width: 812px) and (max-width: 899px) {
+  .tesla-bgd {
+    width: 100%;
+    top: -35px;
+    left: 0;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 811px) {
+  .tesla-bgd {
+    top: -22px;
+    max-width: 831px;
+    left: -12px;
+  }
+}
+
+@media (min-width: 700px) and (max-width: 767px) {
+  .tesla-bgd {
+    top: -22px;
+    width: 100%;
+    height: auto;
+  }
+  .tesla-car {
+    max-width: 700px;
+    height: auto;
+    padding: 80px 0 30px 0;
+  }
+  .content__data-car {
+    flex-direction: column;
+    padding: 11px 0 55px 80px;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 699px) {
+  .tesla-bgd {
+    top: -22px;
+    width: 100%;
+    height: auto;
+  }
+  .tesla-car {
+    max-width: 600px;
+    height: auto;
+    padding: 69px 0 30px 0;
+  }
+  .content__data-car {
+    flex-direction: column;
+    padding: 11px 0 55px 45px;
+  }
+}
+
+@media (min-width: 568px) and (max-width: 599px) {
+  .tesla-bgd {
+    top: -22px;
+    max-width: 100%;
+    height: auto;
+  }
+  .tesla-car {
+    max-width: 580px;
+    height: auto;
+    padding: 57px 0 30px 0;
+  }
+  .content__data-car {
+    flex-direction: column;
+    padding: 11px 0 55px 25px;
+  }
+}
+
+@media (min-width: 568px) and (max-width: 767px) {
+  .menubar__link {
+    display: none;
+  }
+  .content__main-title {
+    font-size: 75px;
+  }
+  .content__facts {
+    padding: 0 0 54px 0;
+  }
+  .data__speed-block {
+    padding-right: 92px;
+  }
+  .data__speed-icon {
+    width: 55px;
+    height: 45px;
+    padding: 7px 14px 0 0;
+  }
+  .content__road-icon {
+    width: 40px;
+    height: 58px;
+  }
+  .data__dimention {
+    align-items: center;
+  }
+  .data__figures {
+    font-size: 43px;
+    padding-right: 6px;
+  }
+  .testdrive__img {
+    display: none;
+  }
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color:#e51937;
+  }
+  .testdrive__text:before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color:#e51937;
+    color: #fff;
+    font-size: 16px;
+  }
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)